.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ on:
       python_versions:
         required: false
         type: string
-        default: '["3.7"]'
+        default: '["3.8"]'
```

Comment:
@PProfizi Minor: just that you know, I've looked for "3.7" references into the repository and it looks we still have a few Python 3.7 references

![image](https://github.com/ansys/pydpf-core/assets/74727146/a0075124-2458-4ff7-a0f6-a7f52f7bbf01)

Reply:
@anslpa the problem I have with removing these options from the dropdown lists is that although we do not support 3.7 anymore for our latest packages versions, people may still want to report issues or questions about an older package which still supported it.